plugins/blockvault_client/connect_mgr.cpp

```diff
@@ -0,0 +1,105 @@
+#include <boost/beast/core.hpp>
+#include <boost/beast/http.hpp>
+#include <boost/beast/ssl.hpp>
+#include <boost/beast/version.hpp>
+#include <boost/asio/connect.hpp>
+#include <boost/asio/ip/tcp.hpp>
+#include <boost/asio/ssl/error.hpp>
+#include <boost/asio/ssl/stream.hpp>
+#include <cstdlib>
+#include <iostream>
+#include <string>
+#include <fstream>
+#include <mutex>
+#include <regex>
+#include <condition_variable>
```

Comment:
I would prefer there to be a logical ordering these includes; personal preference.